cmp/tab/TabPaneModel.js

```diff
@@ -27,8 +27,8 @@ export class TabPaneModel {
 
     /**
      * @param {string} id - unique ID, used for generating routes.
-     * @param {string} name - display name for the tab.
-     * @param {Component} - component (or React node) to be displayed within the tab.
+     * @param {string} [name] - display name for the tab.
+     * @param {Component} component - component (or React node) to be displayed within the tab.
```

Comment:
This is actually a lie.  Its the class itself, not an instance of it.   See the code below where it is used:

Should be something like
@param {Object} componentClass - class of React Component to be displayed within the tab.